hcn/hcnerrors.go

```diff
@@ -20,6 +20,7 @@ var (
 	errInvalidNamespaceID    = errors.New("invalid namespace ID")
 	errInvalidLoadBalancerID = errors.New("invalid load balancer ID")
 	errInvalidRouteID        = errors.New("invalid route ID")
+	ErrUnsupportedRequest    = errors.New("The specified request is unsupported.")
```

Comment:
Full stop '.' not required at the end of the string,

ErrUnsupportedRequest- does this need to be global variable? Are we using this outside hcsshim, in kubeproxy?

Reply:
It's no more used. Removed